### PR TITLE
[logz-233] Logzio update 

### PR DESCRIPTION
**Pull request by yyyogev**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added a new playbook and minor fixes

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by fvigo** [comment]
Adding @David-BMS as he worked on this Playbook before

**Comment by fvigo** [comment]
@yyyogev  please add a better error handling when timestamp cannot be parsed

image

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 1]
```diff
@@ -0,0 +1,3568 @@
...
+  "69":
+    id: "69"
+    taskid: 89374f69-2a20-48ef-892d-c6c95d49f40e
+    type: regular
+    task:
+      id: 89374f69-2a20-48ef-892d-c6c95d49f40e
+      version: -1
+      name: Logzio search without custom fields
+      description: Runs an Apache Lucene query on your Logz.io Operations account
+      script: Logz.io_copy|||logzio-search-logs
```
Comment (David-BMS) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:852]: Wrong integration.

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 2]
```diff
@@ -0,0 +1,3568 @@
...
+      type: regular
+      iscommand: true
+      brand: Logz.io_copy
```
Comment (David-BMS) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:855]: Wrong integration

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 3]
```diff
@@ -0,0 +1,3568 @@
...
+  "71":
+    id: "71"
+    taskid: 485623c1-a4f6-4563-8557-68c4605599a6
+    type: regular
+    task:
+      id: 485623c1-a4f6-4563-8557-68c4605599a6
+      version: -1
+      name: Logzio search without custom fields
+      description: Runs an Apache Lucene query on your Logz.io Operations account
+      script: Logz.io_copy|||logzio-search-logs
```
Comment (David-BMS) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:889]: Wrong integration

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 4]
```diff
@@ -0,0 +1,3568 @@
...
+      type: regular
+      iscommand: true
+      brand: Logz.io_copy
```
Comment (David-BMS) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:892]: Wrong integration

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 5]
```diff
@@ -0,0 +1,3568 @@
...
+  "72":
+    id: "72"
+    taskid: 5eed3c4d-a5a5-4a02-8534-81c276be88e0
+    type: regular
+    task:
+      id: 5eed3c4d-a5a5-4a02-8534-81c276be88e0
+      version: -1
+      name: Logzio search without custom fields
+      description: Runs an Apache Lucene query on your Logz.io Operations account
+      script: Logz.io_copy|||logzio-search-logs
```
Comment (David-BMS) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:927]: Wrong integration

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 6]
```diff
@@ -0,0 +1,3568 @@
...
+      type: regular
+      iscommand: true
+      brand: Logz.io_copy
```
Comment (David-BMS) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:930]: Wrong integration

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting_README.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+ |  | Optional |
+| LogzioURLDomainField | The name of the fields in Logz.io for which you would like to search the URL\\Domain values
+Multiple fields is available by using comma\-separated  |  | Optional |
+| earliest_time | Earliest time to search.
+Examples:
+\-7d
+\-24h
+
+More examples
+https://docs.splunk.com/Documentation/Splunk/8.0.0/Search/Specifytimemodifiersinyoursearch
```
[the remark this answers is not in the document]
Comment (DorisNaaman) [reply Packs/Logzio/Playbooks/Logzio_Indicator_Hunting_README.md:55]: where can i find it in the system ?

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 8]
```diff
@@ -0,0 +1,3568 @@
...
+    simple: -1d
+  required: false
+  description: |-
+    Latest time to search.
+    Examples:
+    -6d
+    -23h
+
+    More examples
+    https://docs.splunk.com/Documentation/Splunk/8.0.0/Search/Specifytimemodifiersinyoursearch
```
[the remark this answers is not in the document]
Comment (DorisNaaman) [reply Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:3483]: where can i find it in the system ?

[the remark this answers is not in the document]
Comment (liorblob) [reply Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:3483]: @David-BMS I think it's coming from the command description

**Comment by liorblob** [comment]
@yyyogev @DorisNaaman @David-BMS can we schedule a demo for the new use cases?

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml (hunk 8 above)
[the remark this answers is not in the document]
Comment (David-BMS) [reply Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:3483]: So it needs to be fixed on the integration as well

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting_README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (David-BMS) [reply Packs/Logzio/Playbooks/Logzio_Indicator_Hunting_README.md:55]: Line 55 in the readme file

**Comment by David-BMS** [comment]
@liorblob sounds like a plan

File: Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml [hunk 9]
```diff
@@ -1,6 +1,6 @@
-id: Logz.io Indicator Hunting
+id: 'Logz.io Indicator Hunting '
```
Comment (liorblob) [inline Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml:1]: @yyyogev please remove space and quotes

**Comment by liorblob** [comment]
@yyyogev please note that some important components were removed from the playbook. Compare it with the previous version and restore them. Also we're waiting for some additional fixes from @DorisNaaman's side.
@David-BMS fyi

**Comment by yyyogev** [comment]
@liorblob we are aware of that and it's fine.

**Comment by DorisNaaman** [comment]
Hi !

Regarding the fixes
it was something with the parsing on our side
so, we fixed it and now everything goes ok in the context data regarding
the “External\Internal IP’s and the IPV6 that was removed automatically,

the only change i made in the playbook itself, is removing an input field
that we are not using anymore "TypeOfLog"


*Dor Doris Naaman*
Security Product
dor.naaman@logz.io

**Comment by liorblob** [comment]
@yyyogev ok cool, but some descriptions and other things were removed and now the validation is failing. You can run `demisto-sdk format` to fix that.

**Comment by yyyogev** [comment]
@liorblob the format is failing on the file 
` File "/venv/bin/demisto-sdk", line 7, in <module>
    from demisto_sdk.__main__ import console_entry
  File "/venv/lib/python2.7/site-packages/demisto_sdk/__main__.py", line 5
    def console_entry() -> None:
`

**Comment by liorblob** [comment]
@yyyogev how did you run the command? also please attach the full error.

**Comment by yyyogev** [comment]
`
demisto-sdk format -i Packs/Logzio/Playbooks/Logzio_Indicator_Hunting.yml                                                          
Traceback (most recent call last):
  File "/venv/bin/demisto-sdk", line 7, in <module>
    from demisto_sdk.__main__ import console_entry
  File "/venv/lib/python2.7/site-packages/demisto_sdk/__main__.py", line 5
    def console_entry() -> None:
                        ^
SyntaxError: invalid syntax

`

**Comment by liorblob** [comment]
@yyyogev looks like it's running with python 2. try to run with python 3.

**Pull request merged**